.github/workflows/ci.yml

```diff
@@ -92,6 +97,9 @@ jobs:
           # brew update sometimes fails with "Fetching /usr/local/Homebrew/Library/Taps/homebrew/homebrew-cask failed!"
           brew update || brew update-reset
           brew install autoconf automake libtool
+      - name: "Set CC"
+        run: |
+          echo "CC=clang -target ${{ matrix.arch }}-apple-darwin$(uname -r)" >> $GITHUB_ENV
```

Comment:
Can't use `env:` because of uname i guess?

Reply:
Yup, that's the exact reason. The value of `env` is a string instead of being evaluated.